app/javascript/modules/listingDetailsLottery/ListingDetailsLottery.tsx

```diff
@@ -1,59 +1,72 @@
 import { ListingDetailsLotteryModal } from "./ListingDetailsLotteryModal"
+import { localizedFormat, renderInlineWithInnerHTML } from "../../util/languageUtil"
+import ErrorBoundary, { BoundaryScope } from "../../components/ErrorBoundary"
 
 export interface ListingDetailsLotteryProps {
   listing: RailsListing
 }
 
 export const ListingDetailsLottery = ({ listing }: ListingDetailsLotteryProps) => {
-  const [lotteryBucketDetails, setLotteryBucketDetails] = useState<RailsLotteryBucketsDetails>()
+  const [lotteryBucketDetails, setLotteryBucketDetails] = useState<RailsLotteryResult>()
   const [isModalOpen, setIsModalOpen] = useState(false)
 
   useEffect(() => {
-    void getLotteryBucketDetails(listing.Id).then((lotteryBucketDetails) => {
-      setLotteryBucketDetails(lotteryBucketDetails)
-    })
-  }, [listing.Id])
+    if (isLotteryComplete(listing)) {
+      void getLotteryBucketDetails(listing.Id).then((lotteryBucketDetails) => {
+        setLotteryBucketDetails(lotteryBucketDetails)
+      })
+    }
+  }, [listing, listing.Id])
 
   return (
     isLotteryComplete(listing) && (
       <div className="border-b pt-4 text-center">
-        <Heading className="mb-4" priority={4}>
-          {t("listings.lottery.lotteryResults")}
-        </Heading>
-        <p className="mb-4 text-sm uppercase">
-          {dayjs(listing.Lottery_Results_Date).format("MMMM D, YYYY")}
-        </p>
-        <div className="bg-gray-100 py-4">
-          <Button
-            size={AppearanceSizeType.small}
-            styleType={AppearanceStyleType.primary}
-            onClick={() => {
-              setIsModalOpen(true)
-            }}
-          >
-            {t("listings.lottery.viewLotteryResults")}
-          </Button>
-        </div>
+        <ErrorBoundary boundaryScope={BoundaryScope.component}>
+          <Heading className="mb-4" priority={4}>
+            {t("lottery.lotteryResults")}
+          </Heading>
+          <p className="mb-4 text-sm uppercase">
+            {localizedFormat(listing.Lottery_Results_Date, "LL")}
+          </p>
+          <div className="bg-gray-100 py-4">
+            {listing.Lottery_Summary && (
+              <div className="mb-3 mx-2 text-gray-700 text-sm">
+                {renderInlineWithInnerHTML(listing.Lottery_Summary)}
+              </div>
+            )}
+            <Button
+              size={AppearanceSizeType.small}
+              styleType={AppearanceStyleType.primary}
+              onClick={() => {
+                setIsModalOpen(true)
+              }}
+            >
+              {t("listings.lottery.viewLotteryResults")}
+            </Button>
+          </div>
 
-        <ListingDetailsLotteryModal
-          isOpen={isModalOpen}
-          listing={listing}
-          lotteryBucketDetails={lotteryBucketDetails}
-          onClose={() => setIsModalOpen(false)}
-        />
+          {lotteryBucketDetails && (
+            <Modal onClose={() => setIsModalOpen(false)} open={isModalOpen} title="">
```

Comment:
what's the reason for moving the `Modal` out of `ListingDetailsLotteryModal`?

Reply:
There's a lot of dynamic content in the modal content so this made it easier to test without having to keep opening the modal. Perhaps the component in the modal would be better named as `ListingDetailsLotterySearchForm` or something like that?

Reply:
ah, got it! I like the idea of renaming the inner component